---
layout: default
title: "Standigm"
subtitle: "Project"
excerpt: "Standigm currently focuses on predicting new indications for existing drugs, called drug repositioning through deeply trained AI model with molecular features of drug responses and drug uses."
header:
  overlay_image: /assets/images/banner.jpg
  overlay_filter: rgba(0, 100, 100, 0.5)
  overlay_filter_to: rgba(255, 255, 255, 0.5)
---

{% include stan_page__hero.html %}

<!-- Project -->
<div class="box" id="project">

    <div class="row">

        <div class="col-xs-12 col-sm-6">
            <div class="mark">
              <img class="poster" src="/assets/images/poster/p1_thumb.jpg" alt="Standigm Poster1">
                <a href="/assets/pdf/Standigm_P1_AZ_challenge.pdf">
                    <img class="pdf-icon" src="/assets/images/pdf_icon.svg" />
                </a>
            </div>
            <div class="desc">
                <div class="poster-title">Project 1:  Predicting synergetic drug combinations</div>
                <div class="summary">
                    The goal of the project is to predict synergetic drug combinations that
                    effectively kill cancer cells. The disease context, the characteristics of drugs,
                    and the interaction between drugs and diseases are described using various chemical
                    properties of drugs, target proteins/signaling pathways of drugs, monotherapy drug
                    response data and genomic data for cell lines, among others. Based on these data,
                    we designed three types of input features (cell line-specific, drug-specific,
                    and drug-cell line interaction features). With various combinations of these features
                    and different learning parameters, we built an ensemble of gradient boosting classifiers,
                    which ranked 3rd among 71 teams worldwide in "AstraZeneca-Sanger Drug Combination
                    Prediction Dream Challenge (2015~2016)". The methods will be applied to precision
                    medicine, patient stratification, and therapeutic/diagnostic design.
                </div>
                <div class="status">Status: Completed in May 2016</div>

            </div>
            <!-- <div class="bg_div"></div> -->
      </div>

        <div class="col-xs-12 col-sm-6">
            <div class="mark">
                <img class="poster" src="/assets/images/poster/p2_thumb.jpg" alt="Standigm Poster2">
                <a href="/assets/pdf/Standigm_P2_Learning_drug_perturbed_data.pdf">
                    <img class="pdf-icon" src="/assets/images/pdf_icon.svg" />
                </a>
            </div>
            <div class="desc">
                <div class="poster-title">Project 2: Learning drug-perturbed data</div>
                <div class="summary">
                    The goal of the project is to predict new indications for existing drugs,
                    called drug repositioning, from the features of drug responses and drug uses
                    discovered via deep learning. Here, the drug responses and uses are described
                    by the data of drug-perturbed gene expressions and of therapeutic usage labels,
                    respectively. We have been developing various multimodal learning methods
                    (XGBoost, Compact Bilinear Pooling, etc.) for robust feature learning.
                    These models suggest 20 hits with new drug indications. For two of the 20 drug
                    repositioning hits, literature has shown their experimental validations. For the
                    remaining hits, Standigm is designing in-vivo and in-vitro experimental validations
                    to demonstrate Standigm.AI’s prediction power. Our drug repositioning AI will discover
                    hidden relational patterns of drugs and indications, ultimately saving the cost of drug
                    development.
                </div>
                <div class="status">Status: Completed in January 2017</div>
            </div>

        </div>

        <div class="col-xs-12 col-sm-6">
            <div class="mark">
                <img class="poster" src="/assets/images/poster/p3_thumb.jpg" alt="Standigm Poster3">
                <a href="/assets/pdf/Standigm_P3_Building_Learning_biological_knowledge.pdf">
                    <img class="pdf-icon" src="/assets/images/pdf_icon.svg" />
                </a>
                <div class="content"></div>
            </div>
            <div class="desc">
                <div class="poster-title" >Project 3: Building &amp; Learning biological knowledge</div>
                <div class="summary">
                    The goal of the project is to predict missing relationships between proteins, drugs, and diseases by learning from a biological knowledge GraphDB. We constructed the biological GraphDB consisting of nodes (32,373 proteins, 7815 drugs, 3721 diseases, etc.) and edges (55,618 drug-disease edges, 125,686 drug-protein edges, etc.) using knowledge from various open and private sources. Users can easily access and visualize the GraphDB via web browsers and navigate the DB using the Cypher query language. We are developing models to predict unknown links between the nodes (proteins, drugs, disease etc.) in the GraphDB. The models will be applied to finding new drug indications, new drug targets, and new disease biomarkers.
                </div>
                <div class="status">Status: In progress (January 2017)</div>
            </div>
        </div>

        <div class="col-xs-12 col-sm-6">
            <div class="mark">
                <img class="poster" src="/assets/images/poster/p4_thumb.jpg" alt="Standigm Poster4">
                <a href="/assets/pdf/Standigm_P3_Building_Learning_biological_knowledge.pdf">
                    <img class="pdf-icon" src="/assets/images/pdf_icon.svg" />
                </a>
                <div class="content"></div>
            </div>
            <div class="desc">
                <div class="poster-title" >Project 4: Artificial Applicable Intelligence</div>
                <div class="summary">
                    Standigm currently focuses on predicting new indications for existing drugs, called drug repositioning through deeply trained AI model with molecular features of drug responses and drug uses. Standigm generated tens of drug candidates for cancers, Parkinson’s disease, autism, fatty liver diseases (NASH) and more. Our candidates have been experimentally being validated with several collaboration partners, CrystalGenomics, Inc. (cancers, under lead optimisation), school of pharmacy/medicine Ajou university (PD, under validating animal model), Korea institute of science and technology (autism, under validating animal model) and a CRO for NASH (in-house, organ on a chip validation finished).
                </div>
                <div class="status">Status: In progress (August 2017)</div>
            </div>
        </div>

        <div class="col-xs-12 col-sm-6">
            <div class="mark">
                <img class="poster" src="/assets/images/poster/p4_thumb.jpg" alt="Standigm Poster4">
                <a href="/assets/pdf/Standigm_P3_Building_Learning_biological_knowledge.pdf">
                    <img class="pdf-icon" src="/assets/images/pdf_icon.svg" />
                </a>
                <div class="content"></div>
            </div>
            <div class="desc">
                <div class="poster-title" >Project 5: DTI</div>
                <div class="summary">
                    Standigm currently focuses on predicting new indications for existing drugs, called drug repositioning through deeply trained AI model with molecular features of drug responses and drug uses. Standigm generated tens of drug candidates for cancers, Parkinson’s disease, autism, fatty liver diseases (NASH) and more. Our candidates have been experimentally being validated with several collaboration partners, CrystalGenomics, Inc. (cancers, under lead optimisation), school of pharmacy/medicine Ajou university (PD, under validating animal model), Korea institute of science and technology (autism, under validating animal model) and a CRO for NASH (in-house, organ on a chip validation finished).
                </div>
                <div class="status">Status: In progress (August 2017)</div>
            </div>
        </div>
        <div class="col-xs-12 col-sm-6">
            <div class="mark">
                <img class="poster" src="/assets/images/poster/p5_thumb.jpg" alt="Standigm Poster5">
                <a href="/assets/pdf/Standigm_P6_EMR_data_representation.pdf">
                    <img class="pdf-icon" src="/assets/images/pdf_icon.svg" />
                </a>
                <div class="content"></div>
            </div>
            <div class="desc">
                <div class="poster-title" >Project 6: EMR data representation</div>
                <div class="summary">
                    The goal of the project is to develop intelligent methods for patient and medical record representation learning, and ultimately predict clinical events such as patient diagnosis, prognosis, or medication categories.
                    We collaborate with Ajou University School of Medicine, Korea. This research is supported by a grant of the Korea Health Technology R&D Project, “Development of the Artificial Intelligence supporting a Clinical Trial System (AI-CTS)”
                </div>
                <div class="status">Status: In progress (September 2017)</div>
            </div>
        </div>     

    </div>
</div>
